cdap-ui/app/features/adapters/adapters.less

```diff
@@ -14,42 +14,523 @@
+      }
+
+      .modal-content .text-center {
+        margin: 25px auto;
+
+        button.btn {
+          height: 35px;
+          width: 70px;
+          border: 0;
+
+        }
+      }
+    }
+
+
+    .tiles-container {
+      background-color: white;
+      border-radius: 5px;
+
+      a:hover {
+        text-decoration: none;
+      }
+
+      .heading {
+        margin-top: 5px;
+        margin-bottom: 10px;
+        h3 {
+          font-weight: 500;
+          margin-bottom: 0;
+          font-size: 21px;
+          letter-spacing: 0.5px;
+        }
+
+        &.preconfigured {
+          margin-bottom: 50px;
+        }
+      }
+
+      .info {
+        margin-bottom: 35px;
+
+        h4 {
+          font-weight: 400;
+          font-size: 14px;
+          letter-spacing: 0.3px;
+        }
+      }
+    }
+
+    // Studio start screen
+    .hydrator-start {
+      margin-top: 140px;
+    }
+    .hydrator-preconfigured {
+      margin-top: 70px;
+    }
+
+    .templates-filters {
+      .has-feedback > .form-control-feedback {
+        line-height: 32px;
+      }
+
+      .btn-group-vertical {
+        width: 100%;
+        margin: 15px 0;
+
+        button:focus {
+          background-color: #e2e3e5;
+        }
+      }
+    }
+
+    div.template-link {
+      cursor: pointer;
+      width: 100%;
+
+      p {
+        color: lighten(@btn-default-color, 15%);
+        margin-bottom: 2px;
+      }
+
+      &:hover {
+        background-color: lighten(@btn-default-color, 15%);
+        text-decoration: none;
+        .box-shadow(inset 0 0 20px 0 fade(black, 25%));
+
+        h4, p { color: white; }
+      }
+
+      hr {
+        border: 0;
+        height: 0;
+        width: 100%;
+        border-top: 1px solid fade(black, 10%);
+        border-bottom: 1px solid fade(white, 30%);
+        margin: 10px auto;
+      }
+    }
+
+    .start-well {
+      background-color: white;
+      font-weight: 500;
+      margin-right: auto;
+      margin-left: auto;
+      height: 180px;
+      width: 180px;
+      cursor: pointer;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      .border-radius(6px);
+      h4, a {
+        font-weight: 500;
+      }
+      h4 {
+        color: @btn-default-color;
+
+        &.template-name {
+          margin-bottom: 10px;
+        }
+      }
+
+      p {
+        font-size: 12px;
+      }
+
+      &:hover {
+        background-color: lighten(@btn-default-color, 15%);
+        .box-shadow(inset 0 0 20px 0 fade(black, 25%));
+
+        h4 { color: white; }
+      }
+
+    }
+    .canvas-wrapper {
+      position: relative;
+      margin-top: 60px;
+
+      .left-panel-wrapper {
+        .left-panel {
+          height: 100%;
+        }
+      }
+    }
+    my-dag {
+      width: 100%;
+      // Dag-Height 60% on smaller screen
+      height: 60vh;
+      @media (min-height: 800px) {
+        height: 40vh;
+      }
+    }
+
+    .tab-pane .panel-default {
```

Comment:
There was only one style block that needed to be kept, and since it's tied to validators, I've moved it to validators.less.